Dockerfile

```diff
@@ -1,16 +1,29 @@
 # vim:set ft=dockerfile:
-FROM vertexproject/synapse-base-image:py36
 
-ENV SYN_DMON_LOG_LEVEL="WARNING"
+# Example Usage
+#
+# To Build:
+#   docker build -t vertexproject/synapse:localdev .
+#
+# To Run a example Cortex on port 47322:
+#   docker run -it -p47322:47322 vertexproject/synapse:localdev
+#
+# To create and run your own dmon (do not create the dmon dir inside synapse dir)
+#   python -m synapse.tools.deploy --listen tcp://0.0.0.0:47322 cortex core dmon_dir
+#   docker run -it -p47322:47322 -v "$(pwd)"/dmon_dir:/syndata/dmon_dir vertexproject/synapse:localdev
+
+FROM vertexproject/synapse-base-image:py36
 
-COPY . /root/git/synapse
-RUN mkdir /syndata \
- && cd /root/git/synapse && python setup.py install \
- && cp synapse/docker/cortex/ram_dmon.json /syndata/dmon.json
+COPY . /root/git/synapse/
+RUN \
+    # Install Synapse
+    cd /root/git/synapse && \
+    python setup.py develop && \
+    # Create a default dmon directory
+    mkdir -p /syndata && cd /syndata && \
+    python -m synapse.tools.deploy --listen tcp://0.0.0.0:47322 cortex core dmon_dir
```

Comment:
I keep thinking about this and how, if a user maps in a directory to `/syndata/dmon_dir` with an additional dmon/cell config, they're still going to have the cell config for the cortex named `core` and that guy will always be running by default.  I wonder if there is a balance we can strike by having a default entrypoint script that can work to spin up a cortex, but is trivially smashable without alot of work by a end user. Otherwise, either we have to drop a new dmon dir and execute that dir with a custom entry point; or do a explicit removal of the default core with custom images every time.

Reply:
If you map in your own volume, it will smash the contents of the existing default volume, so you won't end up with two.

If you don't map anything in, you get a default cortex. If you do want to run something else, either put it in that dir, or change the entrypoint.

This works the same way it does now (by default it runs a ram core, if you want it to do something else, you have to map in a file to `/syndata/dmon.json` or you have to specify the entrypoint)

Reply:
> If you map in your own volume, it will smash the contents of the existing default volume, so you won't end up with two.

ok - for some reason i thought it was additive. in which case, tally ho!